lib/components/molecules/QuoteMedia/QuoteMedia.style.js

```diff
@@ -1,26 +1,16 @@
 import { css } from 'styled-components';
 
 const styles = css`
-  display: grid;
-  .column {
-    grid-template-columns: repeat(2, 1fr);
-  }
-  .stacked {
-    grid-template-columns: repeat(1, 1fr);
-  }
-  .title {
+  .title,
+  .media {
     ${props => `
-        ${props.theme.typography.FONT_H2};
-        padding: ${props.theme.spacing.unit2};
+      padding: ${props.theme.spacing.unit2};
     `};
-    grid-column: 1/-1;
   }
-  .media {
-    ${props => `
-        padding: ${props.theme.spacing.unit2};
-  `};
+  .container {
+    display: flex;
+    flex-direction: ${props => (props.variation === 'stacked' ? 'column' : 'row')};
```

Comment:
If the variation prop is stacked then flex direction will be column. otherwise row.